ip-messaging/rest/accounts/list-get-example-2/meta.json

```diff
@@ -0,0 +1,4 @@
+{
+  "title": "List all active accounts",
```

Comment:
Title case, just marking them all so you can see all the commetns gone after the next commits